packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -424,8 +424,8 @@ export class AggregatedAttestationPool {
     return attestations;
   }
 
-  private onScrapeMetrics({opPool}: Metrics): void {
-    const metrics = opPool.aggregatedAttestationPool;
+  private onScrapeMetrics(metrics: Metrics): void {
```

Comment:
@twoeths reverting some changes I pushed to your branch before to align metric variable names a bit in which case the abbreviation I did before it not great, so the verbose variable name you used seems better